tools/link/bin/link.sh

```diff
@@ -28,18 +28,18 @@ for d in *; do
     fi
 done
 cd $LEAFYGREEN_HOME
-yarn
+pnpm
 set +e
-yarn run link:all
+pnpm run link:all
 set -e
 cd $LEAFYGREEN_HOME
 # Add `--no-build` to the end of the command to skip the build step
 if [$2 != "--no-build"]; then
-    yarn build
+    pnpm build
 fi
 cd $APPLICATION_HOME
 for f in "${INSTALLED_PACKAGES_ARRAY[@]}"; do
-    yarn link @leafygreen-ui/$f
+    pnpm link @leafygreen-ui/$f
```

Comment:
not entirely sure this is a 1:1 api change

Reply:
Actually, we have a ticket to update this. It's fine for now
 https://jira.mongodb.org/browse/LG-3475